src/styles/abstracts/_variables.scss

```diff
@@ -2,10 +2,48 @@
  * Variables used as standards throughout The Gazelle
  */
 
-// Colors
+/*
+ * Colors
+ */
 $dark-purple:	#433362;
 $dark-grey: #595959;
 $border-grey: #C6C6C6;
 
-// Media Queries
-$small: 750px;
+/*
+ * Media Queries
+ */
+$medium: 780px;
+$small: 540px;
+
+
+/*
+ * Grid
+ */
+// 3 column grid
+  // Single column
+  $column-3-1: 30%;
```

Comment:
Little confused by this naming here. What does 3-1 mean?

Also are you doing the columns by % or by flexbox?